### ISSUE TITLE
CLI: Unicode passwords handling is broken

### ISSUE THREAD

**Issue by Kuurusch** [keepassxreboot/keepassxc#8305]
## Overview
So, just found out, that the cli on windows does not escape password correctly! Don't know if there other characters, but at least 'ç' does not work! So its definitly a bug. 

## Steps to Reproduce
Try to create new database with the cli-version of keepassXC
When it promts me to type in password, everything works as acpected. 
But when I want to open afterwards the database over the GUI, it does not work. When I try to create a database with a password which does not contain the caracter 'ç', everything works!

**Comment by louib** [comment]
@Kuurusch how about when you try to open the database with the cli after creating it? You could try a simple `db-info` for example.

**Comment by droidmonkey** [comment]
Windows does not use UTF-8 encoding in cmd.exe by default. You need to set your code page for this to work (sometimes it still won't work). This is outside of our control. 

See here: https://newbedev.com/using-utf-8-encoding-chcp-65001-in-command-prompt-windows-powershell-windows-10

Also in the gui, in order to type special characters with dead keys you need to UNHIDE the password. This is a known Qt bug. So that might also be your problem.

**Comment by Kuurusch** [comment]
@droidmonkey Yea, I've changed the code-page. I've did for example the same with Veracrypt (TrueCrypt) and there everything works perfectly. Firstly I had also the problem, that the pipe under Windows introduces a space after an echo-command, which was quite ennoying. But I've found a way around this problem. Because under UNIX I do just printf password\npassword | keepass-cli ..... But this of corse does not work under Windows.

But it could be a problem with this Qt-bug, I will check!

**Comment by Kuurusch** [comment]
@droidmonkey @louib hm, so I've could'nt reproduce this Qt bug. I had no luck to open the db over the GUI, even with showing the Password. But what I've recognised, when I tipe in the password with for example an 'ä', I have then to press two times the enter-key to go further. This does not happen, if I enter only asccii-characters. So it seams, that this special character puts the terminal in some other mode? So I would say, under Windows the password input in the console is completly broken! (I've changed for this tests the codepage for utf8 => chcp 65001)
But of corse, it works to open the db in the cosole e.g. db-info
